src/api/app/views/webui2/webui/groups/_group_members.html.haml

```diff
@@ -25,11 +25,10 @@
                   class: 'group-member-removal float-right', title: 'Remove member from group') do
                   %i.fas.fa-sm.fa-times-circle.text-danger
               .card-body.ml-2
-                %b
-                  = link_to(user_show_path(user)) do
-                    = display_name(user)
+                = link_to(user_show_path(user), class: 'user-realname') do
+                  = display_name(user)
```

Comment:
I wonder if we should split the name and the login. Could one of them be in italic for example?

Reply:
Sometimes we only show the name, sometimes name and login... and introducing some italic text here... :thinking: I would say it's going to be less homogeneous than it was before.

Reply:
Alright, no worries.

Reply:
Wait, I assumed you wanted to split in different lines. I gave it a try with italic but in the same line and I like it. What do you think @dmarcoux?

![Screenshot-2019-8-20 Open Build Service(4)](https://user-images.githubusercontent.com/2581944/63338207-aa10db00-c342-11e9-932a-984ca316fec8.png)

It also looks nice in
![Screenshot-2019-8-20 Open Build Service(3)](https://user-images.githubusercontent.com/2581944/63338233-b432d980-c342-11e9-9767-9025de3764a8.png)

Reply:
I like it :+1: !